data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,584 @@ mission "Coalition License Patch"
+	stopover "Second Viridian"
+	stopover "Remote Blue"
+	on offer
+		conversation
+			`This time you're being expected, and are brought to a large conference room where many Saryds, and the occasional Arach, are waiting. A Kimek that is part of the team organizing the containment procedures comes in, and says you will once again be transporting vaccines.`
+			`	"In a few hours, fully replicated, the vaccine samples will be. Loaded onto your ships, they will be. Delivered to <stopovers>, they must be."`
+			`	Everyone is then separated into groups, and you're put with two Saryd captains, and an Arachi who is comically wearing some protective suit of sorts that covers their entire body, making it somewhat difficult for him to walk.`
+			`	The captains agree to follow your lead and deliver the vaccines along with you, so you all head back to your ships.`
+				accept
+	npc accompany save
+		personality escort timid
+		government "Coalition"
+		ship "Saryd Sojourner" "Noset Temu"
+		ship "Saryd Sojourner" "Miest Laba"
+		ship "Arach Hulk" "Bloobrupblot"
+	on stopover
+		dialog `You and the Coalition captains have delivered the final batch of vaccines. You must now return to <origin>.`
+	on visit
+		dialog `You have reached <planet>, but you're missing something! Either you haven't visited <stopovers>, or you left the Coalition ships behind.`
+	on complete
+		payment 2593230
+		"coalition jobs" ++
+		conversation
+			`You and the other captains are told to wait in one of several lines formed here. As they talk with others and then with you, they mention how many of the captains here just came back from delivering vaccines to Arachi and Saryd space, to the travelers who got infected before going to planets there.`
+			`	The line moves quickly, and once it's your turn, you're paid <payment>.`
+			`	Looking back at the line, and seeing the sheer number of captains waiting for their share, it's no wonder the Kimek that paid you were extremely brief, but as you come back to your ship, you see that a message was sent to you by the Coalition government. More specifically, it appears to have been sent by their Department of Health.`
+			`	"To all captains who helped in dealing with the recent viral outbreak, you have our gratitude, and may rest tonight knowing you've contributed to the continued prosperity of our Coalition."`
+
+
+
+mission "Saryd Students 1"
+	minor
+	name "Acquire Arachi Rum"
+	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi Rum, currently only sold on <destination>."
+	deadline
+	to offer
+		has "Coalition: First Contact: done"
+	source "Shadow of Leaves"
+	destination "Weir of Glubatub"
+	on offer
+		conversation
+			`A young Saryd bumps into you as you're passing by a nearby shop, knocking you to the ground.`
+			`	They help you up, apologizing. A couple that were with him entered the shop without even noticing you, but almost immediately come back out.`
```

Comment:
this actually refers to a couple other Saryd students, not a couple as in the relationship term. adding "others" after the "couple" to make it clear, since I forgot apparently